test/e2e/e2e_test.go

```diff
@@ -129,7 +144,7 @@ func startCatalyst(t *testing.T, ctx context.Context, hostname, network string)
 			Target:   "/config",
 			ReadOnly: true},
 		},
-		Cmd: []string{"MistController", "-c", fmt.Sprintf("/config/%s.json", hostname)},
+		Cmd: []string{"MistController", "-c", fmt.Sprintf("/config/%s", mcFile)},
```

Comment:
Technically it's mounted into the container. I think copying would be more difficult, because you can only copy a file to a container that is already running.